utils/runtime-fuzzer/src/gear_calls.rs

```diff
@@ -418,6 +418,7 @@ fn config(
             (SysCallName::Exit, 0..=1),
             (SysCallName::Alloc, 3..=6),
             (SysCallName::Free, 3..=6),
+            (SysCallName::FreeRange, 1..=3),
```

Comment:
@SabaunT please check this place

Reply:
I'd remove that, until define some proper config for params. It will still generate free_range, but only once.